content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview.md

```diff
@@ -29,3 +29,13 @@ Tip: Your image should be a PNG, JPG, or GIF file under 1 MB in size. For the be
     - To remove an image, click **Remove image**
 
     ![Social preview dropdown](/assets/images/help/repository/social-preview.png)
+
+## Transparency
+
+PNGs with transparency are supported. Many communication platforms support a dark mode, so using a transparent social preview may be beneficial. The transparent image below is acceptable on a dark background; however, this is not always the case. When using an image with transparency keep in mind how it may look on different color backgrounds or platforms that don't support transparency.
```

Comment:
Slightly rewording + avoiding passive voice + putting the advice given in the last sentence in its own paragraph to highlight it more.

```suggestion
We support PNG images with transparency. Many communication platforms support a dark mode, so using a transparent social preview may be beneficial. The transparent image below is acceptable on a dark background; however, this may not always be the case. 

When using an image with transparency, keep in mind how it may look on different color backgrounds or platforms that don't support transparency.
```